src/misc/ListModel.ts

```diff
@@ -542,6 +542,10 @@ export class ListModel<ElementType extends ListElement> {
 		return this.state.items.length === 0 && this.state.loadingStatus === ListLoadingState.Done
 	}
 
+	isSelectionEmpty(): boolean {
+		return this.state.selectedItems.size === 0
```

Comment:
is this the right thing to check? Shouldn't we check multiselect mode instead?

Reply:
if we open something and then go back to the list isn't it still selected?

Reply:
I think we can´t leave the page when the ```multiSelect``` is on before exiting the multi selection. Also, I just realized that we can start a multi selection with 0 items and checking for ```selectedItems === 0``` will make the app go back instead of disabling the multi selection